core/src/main/scala/org/apache/spark/rpc/netty/Outbox.scala

```diff
@@ -215,7 +216,7 @@ private[netty] class Outbox(nettyEnv: NettyRpcEnv, val address: RpcAddress) {
     // update messages and it's safe to just drain the queue.
     var message = messages.poll()
```

Comment:
Exception creation can be done here instead of inside the loop, keeping usable stack trace